src/App/Containers/BooksList.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+// import PropTypes from 'prop-types'
+import { useSelector, useDispatch } from 'react-redux';
+// import { bookAction } from '../../Redux/actions/index';
```

Comment:
- [ ]  [REQUIRED] Inline comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history